src/kfactory/kcell.py

```diff
@@ -1315,6 +1312,16 @@ def ymax(self) -> int:
         """Returns the x-coordinate of the left edge of the bounding box."""
         return self._kdb_cell.bbox().top
 
+    @property
+    def ysize(self) -> int:
+        """Returns the x-coordinate of the left edge of the bounding box."""
+        return self._kdb_cell.bbox().top - self._kdb_cell.bbox().bottom
+
+    @property
+    def xsize(self) -> int:
+        """Returns the x-coordinate of the left edge of the bounding box."""
+        return self._kdb_cell.bbox().left - self._kdb_cell.bbox().right
+
```

Comment:
bbox has width() and height(), no need to do the math yourself. Also, is it really necessary to alias all of these? Because bbox has the nicety you can do `c.bbox(LAYER.WG).width()` to get the width on WG only
or we build it in directly?

```python
    @property
    def xsize(self, layer: LayerEnum|int|None = None) -> int:
        """Returns the x-coordinate of the left edge of the bounding box."""
        if layer:
            return self._kdb_cell.bbox(layer).width()
        else:
            return self._kdb_cell.bbox().width()
```

Reply:
great!